cvat-ui/src/components/header/styles.scss

```diff
@@ -34,8 +34,6 @@
 
     img {
         object-fit: contain;
-        width: 100%;
```

Comment:
It seems that without a static container width and width: 100% on the image, object-fit: contain doesn't work as expected. As a result, logos larger than 128px won't be scaled down to fit but will be clipped instead.
![image](https://github.com/user-attachments/assets/c915d2aa-f45f-4327-ac75-b45434f511ba)

Reply:
Added max-width, max-height css props

Reply:
However it breaks the implementation.

Reply:
@klakhov 
Try now